cmd/cmp.go

```diff
@@ -15,8 +13,10 @@ var cmpCmd = &cobra.Command{
 	Short:                 "Compare two query result directories",
 }
 
+var defaultRegex = `.*(query_\d{2})(?:_c0)?\.output`
```

Comment:
```suggestion
const defaultRegex = `.*(query_\d{2})(?:_c0)?\.output`
```
Sine you do not need to change it.